src/bundle/Core/Command/UpdateTimestampsToUTCCommand.php

```diff
@@ -121,7 +121,7 @@ protected function configure()
 The command <info>%command.name%</info> updates field
 data_int in configured Legacy Storage database for a given Field Type.
 
-This is to be used either when migrating from eZ Publish to eZ Platform 
+This is to be used either when migrating from Ibexa to Ibexa 
 (when using platform backend instead of legacy), or when upgrading legacy 
 to v2019.03 which has been adapted to use UTC.
```

Comment:
The statement here was valid. If we want to get rid of eZ references then we can maybe rephrase it, sth like:
```suggestion
This is to be used when upgrading from a legacy version which was not adapted to use UTC.
```